pronto_mia/lib/core/factories/analyzed_error_factory.dart

```diff
@@ -36,11 +36,11 @@ class AnalyzedErrorFactory {
       analyzedError.isUnknownError = true;
     }

```

Comment:
Just add a opt-out return then many of the checks afterwards will be obsolete.
```suggestion
      return;
    }

```